docs/external-etcd.md

```diff
@@ -0,0 +1,67 @@
+# Support for external etcd
+
+Cluster API Bootstrap Provider Kubeadm now supports using an external etcd cluster for your workload Kubernetes
+clusters.
+
+To use this you will need to create an etcd cluster and generate an apiserver-etcd-client key/pair.
+[`etcdadm`](https://github.com/kubernetes-sigs/etcdadm) is a good way to get started if you'd like to test this
+behavior.
```

Comment:
Should there be some warnings that:
- The external etcd cluster will be unmanaged and users will have to manage the lifecycle of those external etcd clusters appropriately
- The use of external etcd could incur additional data transit costs if deploying to a cloud provider depending on the provider configuration used

Reply:
Yes, better to provide these warnings now